concourse/steps/replicate_secrets.mako

```diff
@@ -13,8 +13,9 @@ job_mapping_name = extra_args['job_mapping_name']
 secrets_repo_url = extra_args['secrets_repo_url']
 compliance_reporting_repo_url = extra_args['compliance_reporting_repo_url']
 delivery_endpoints_cfg_name = extra_args['delivery_endpoints_cfg_name']
-github_issue_template = extra_args['github_issue_template']
+github_issue_template_cfgs = extra_args['github_issue_template_cfgs']
```

Comment:
since we are back to passing-in a mapping, revert the change previously introduced for passing-in a single template.